<p>Result of the MALDI-TOF analysis can be up-loaded as ASCII files or the mass pick list can be introduced manually in the search engine for identification of the protein of interest. Computer searching algorithms are used to identify protein sequences using peptide mass information (mass map) obtained from proteolytic digests of proteins. If the database does not contain the sequence of the unknown protein, then a successful search will provide those entries showing the closest sequence homology, often equivalent proteins from related species. In order to obtain a satisfactory result multiple parameters required to be specified. The brief definitions of the parameters are given below.</p>
<p>1.&nbsp;<strong>Database:</strong>&nbsp;Database is a systemic organization of data to pile up and retrieve the data for easy accession during the identification the proteins.</p>
<p>2.&nbsp;<strong>Taxonomy:</strong>&nbsp;Taxonomy is an art of classification of organisms based on pre-determined criteria to simplify the search to limited hits.</p>
<p>3.&nbsp;<strong>Fixed modifications:</strong>&nbsp;Some reagents/chemicals have ability to modify the amino acids during sample preparations needs to be specified to reimburse the mass of the modifications while searching for identification.</p>
<p>4.&nbsp;<strong>Variable modifications:</strong>&nbsp;Quite a few of the amino acids (like Met, Tyr) are highly prone to modifications during sample preparation is need to be specified to obtain superior results.</p>
<p>5.&nbsp;<strong>Data format:</strong>&nbsp;Data file which is compactable to search engine having mass peak list, intensity of the peak and search parameters.</p>
<p>6.&nbsp;<strong>Enzyme:</strong>&nbsp;An enzyme has a specificity to cleave the proteins to smaller peptides need to be specified to get the finger prints of whole protein in mass spectrometric analysis.</p>
<p>7.&nbsp;<strong>Allow up to- Miss-cleavage:</strong>&nbsp;Sometime enzyme misses the cleavage site in protein need to be addressed to identify the best hit.</p>
<p>8.&nbsp;<strong>Monoisotopic mass:</strong>&nbsp;Monoisotopic mass indicates the total mass of the highly abundant isotopes of the amino acid.</p>
<p>9.&nbsp;<strong>Average mass:</strong>&nbsp;Average mass is the mass of the average of the all existing isotopes in amino acid.</p>
<p>10.&nbsp;<strong>Instrument:</strong>&nbsp;Quite a few of mass spectrometry instruments are in practice for protein identification with different mass analyzers, ion source and detectors need to be specified to get best result.</p>
<p>11.&nbsp;<strong>Data file:</strong>&nbsp;Data file act as a browser to import dataset generated from mass spectrometry into the search engine to obtain the results.</p>
<p><strong>Search engines:</strong>&nbsp;Different search engines are available for protein identification using mass spectrometric peptide mapping information. MASCOT, SEQUEST and ProFound are the most commonly used search engines for identification of proteins and peptides.</p>
<p><strong>MASCOT</strong>&nbsp;<a href="http://www.matrixscience.com/">(http://fields.scripps.edu/sequest)</a>: SEQUEST is a registered trademark of the University of Washington. This search engine correlates uninterpreted tandem mass spectra of peptides with amino acid sequences from protein and nucleotide databases. The result offers the amino acid sequence and thus the protein(s) and organism(s) that correspond to the mass spectrum being analyzed.</p>
<p><strong>SEQUEST</strong><a href="http://fields.scripps.edu/sequest/">(http://fields.scripps.edu/sequest)</a>: SEQUEST is a registered trademark of the University of Washington. This search engine correlates uninterpreted tandem mass spectra of peptides with amino acid sequences from protein and nucleotide databases. The result offers the amino acid sequence and thus the protein(s) and organism(s) that correspond to the mass spectrum being analyzed.</p>
<p><strong>ProFound</strong><a href="http://prowl.rockefeller.edu/prowlcgi/profound.exe">(http://prowl.rockefeller.edu/prowlcgi/profound.exe)</a>: ProFound uses Bayesian algorithm to identify proteins from protein databases. One of the major advantages of this search engine is that it provides accurate identities of proteins even when the data quality is relatively low or the target protein is present in a complex mixture.</p>
<p></p>