<p><strong>References&nbsp;</strong>:</p>
<h4>Web Links:</h4>
<p>1. http://www.matrixscience.com/</p>
<p>2. http://fields.scripps.edu/sequest/</p>
<p>3. http://prowl.rockefeller.edu/prowl-cgi/profound.exe</p>
<h4>Journals:</h4>
<p>1. Zhang W et al. ProFound: an expert system for protein identification using mass spectrometric peptide mapping information. Anal Chem. 2000, 72, 2482-2489.</p>
<p>2. Cottrell JS. Protein identification by peptide mass fingerprinting. Pept Res. 1994, 7, 115-24.</p>
<p>3. Hirosawa M et al. MASCOT: multiple alignment system for protein sequences based on three-way dynamic programming. Comput Appl Biosci. 1993, 9, 161-167.</p>
<p>4. Perkins DN et al. Probability-based protein identification by searching sequence databases using mass spectrometry data. Electrophoresis. 1999, 20, 3551-3567.</p>
<p>5. Colinge J et al. OLAV:Towards high-throughput tandem mass spectrometry data: Proteomics. 2003, 3, 1454-1463.</p>
<p>6. Egelhofer V et al. Protein identification by MALDI-TOF-MS peptide mapping: a new strategy. Anal Chem. 2002, 74, 1760-1771.</p>
<p>7. K&ouml;cher T et al. Altered Mascot search results by changing the m/z range of MS/MS spectra: analysis and potential applications. Anal Bioanal Chem. 2010, PMID: 21161510.</p>
<p></p>