<p><strong>References&nbsp;</strong>:</p>
<p>1) Singh S et al, Prediction and analysis of paralogous proteins in Trichomonas vaginalis genome, BIOINFORMATION, Volume 6(1).</p>
<p>2) Lancia N.F et al, Proteome analysis of the leukocytes from the American alligator (Alligator mississippiensis) using mass spectrometry, Comparative Biochemistry and Physiology, Part D 5 (2010) 308&ndash;316.</p>
<p>3) Panagiotis D et al, GPU-BLAST: using graphics processors to accelerate protein sequence alignment, BIOINFORMATICS, Vol. 27 no. 2 2011, pages 182&ndash;188 doi:10.1093/bioinformatics/btq644.</p>
<p>4) Dieterich C, Department of Evolutionary Biology, Max Planck Institute for Developmental Biology. BLAST and FASTA Heuristics in pairwise sequence alignment. www.bioalgorithms.info</p>
<p>5) Blast Program Selection Guide http://blast.ncbi.nlm.nih.gov/Blast.</p>
<p>6) Title: Essential Bioinformatics Author: Jin Xiong Publisher: Cambridge University Press Publication Date: 2006-03-13, Chapter three- Sequence Alignment.</p>
<p><strong>Links OF Tools</strong>:</p>
<h2>1.MUSCLE:&nbsp;<a href="http://www.ebi.ac.uk/Tools/msa/muscle/">http://www.ebi.ac.uk/Tools/msa/muscle/</a></h2>
<h2>2.T-COFFEE&nbsp;<a href="http://tcoffee.crg.cat/">http://tcoffee.crg.cat/</a></h2>
<h2>3.COBALT:&nbsp;<a href="http://www.ncbi.nlm.nih.gov/tools/cobalt/">http://www.ncbi.nlm.nih.gov/tools/cobalt/</a></h2>
<h2>4.CLUSTALW2&nbsp;<a href="http://www.ebi.ac.uk/Tools/msa/clustalw2/">http://www.ebi.ac.uk/Tools/msa/clustalw2/</a></h2>
<p></p>