<p>Identification of similar regions in protein sequences to determine the functional, structural and evolutionary relationships between them.</p>
<p>2</p>
<p><strong>Step by step Procedure</strong>:</p>
<p>3</p>
<ol start="1" type="1">4
<li>Choose the type of alignment to be performed whether pairwise or multiple sequence.</li>
5</ol>
<p>6</p>
<ol start="2" type="1">7
<li>Input the subject protein sequence of your choice or upload it through the browse tab in the FASTA format.</li>
8</ol>
<p>9</p>
<ol start="3" type="1">10
<li>Input the query protein sequence or upload it using the browse tab in FASTA format.</li>
11</ol>
<p>12</p>
<ol start="4" type="1">13
<li>Select the general parameters like Max target sequences to align maximum length of a sequence in a row, word size, expected threshold and query matches for pairwise alignment. Select protein weight matrix, Clustering method and file formats for multiple sequence alignment. Gap opening and extension should be kept constant.</li>
14</ol>
<p>15</p>
<ol start="5" type="1">16
<li>Select the scoring parameters. Select the matrix to assign scores for aligning possible pair of residues. Select BLOSUM or PAM for stronger and weaker alignments or distantly related sequences respectively.</li>
17</ol>
<p>18</p>
<ol start="6" type="1">19
<li>Select gap costs so as to decrease the number of gaps in the alignment.</li>
20</ol>
<p>21</p>
<ol start="7" type="1">22
<li>By default, "low complexity region" is selected to filter out the repeated region and improve the sensitivity of sequence similarity search.</li>
23</ol>
<p>24</p>
<ol start="8" type="1">25
<li>Hit the align tab to see the result.</li>
26</ol>
<p>27</p>
<center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/3.png" alt="" /></center>
<p>28</p>
<p>Page composed with the free <a href="https://htmlg.com/html-editor/">online HTML editor website</a>. Please subscribe for a license to remove this message.</p>
<p>Clean HTML - AAA + Case Converter 1810 Cleaning options Remove tag attributes Remove inline styles Remove classes and IDs Remove all tags Remove successive &nbsp;s Remove empty tags Remove tags with one &nbsp; Remove span tags Remove images Remove links Remove tables Replace table tags with structured</p>
<div>s Remove comments Encode special characters Set new lines and text indents Color Picker #BBD0E0 &raquo;
<p>Identification of similar regions in protein sequences to determine the functional, structural and evolutionary relationships between them.</p>
<p><strong>Step by step Procedure</strong>:</p>
<ol start="1" type="1">
<li>Choose the type of alignment to be performed whether pairwise or multiple sequence.</li>
</ol><ol start="2" type="1">
<li>Input the subject protein sequence of your choice or upload it through the browse tab in the FASTA format.</li>
</ol><ol start="3" type="1">
<li>Input the query protein sequence or upload it using the browse tab in FASTA format.</li>
</ol><ol start="4" type="1">
<li>Select the general parameters like Max target sequences to align maximum length of a sequence in a row, word size, expected threshold and query matches for pairwise alignment. Select protein weight matrix, Clustering method and file formats for multiple sequence alignment. Gap opening and extension should be kept constant.</li>
</ol><ol start="5" type="1">
<li>Select the scoring parameters. Select the matrix to assign scores for aligning possible pair of residues. Select BLOSUM or PAM for stronger and weaker alignments or distantly related sequences respectively.</li>
</ol><ol start="6" type="1">
<li>Select gap costs so as to decrease the number of gaps in the alignment.</li>
</ol><ol start="7" type="1">
<li>By default, "low complexity region" is selected to filter out the repeated region and improve the sensitivity of sequence similarity search.</li>
</ol><ol start="8" type="1">
<li>Hit the align tab to see the result.</li>
</ol><center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/3.png" alt="" /></center>
<p>Page composed with the free <a href="https://htmlg.com/html-editor/">online HTML editor website</a>. Please subscribe for a license to remove this message.</p>
</div>