<p>Download the FASTA format of the protein whose details are to be obtained.<br />Open the Gene Ontology home page link provided: URL:<a href="http://www.geneontology.org/GO.tools.shtml">&nbsp;http://www.geneontology.org/GO.tools.shtml</a></p>
<p><br /><br /></p>
<div>&nbsp;<img src="http://pe-iitb.vlabs.ac.in/images/exp15/1.png" alt="" width="690px" height="497px" /></div>
<p><br /><br /></p>
<p>Select the tools according to the information required, there are various category available under the consortium tool heading.</p>
<p><strong>The following are some examples of the tools used for annotation-</strong>:</p>
<ol start="1" type="1">
<li><strong>g:Profiler-</strong>&nbsp;This is a public web server that can characterize and manipulate gene lists from high-throughput genomic data. g:Profiler has a simple user-friendly web interface with powerful visualization. g:Profiler currently supports 85 species, including mammals, fungi, plants, insects, etc, from the Ensembl and Ensembl Genomes databases.</li>
</ol>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/2.png" alt="" /></div>
<p><br /><br /></p>
<p>The result displayed contains the information of the entry input and also the statistical information of the same.</p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/3.png" alt="" /></div>
<ol start="2" type="1">
<li><strong>InterProScan-&nbsp;</strong>It is a Perl-based annotation editor that combines these different methods for protein domain and family recognition into one resource.&nbsp;<br />Open the main page of the tools and paste your FASTA format of the protein.</li>
</ol>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/4.png" alt="" /></div>
<p><br /><br /></p>
<p>The following output is seen in the tool (i) Figure 1: Visual Output &ndash; This displays the various functional domains present in the protein. (ii) Figure 2: Tool Output - Defines the proteins family and super family.</p>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/5.png" alt="" />&nbsp;Figure 1: Visual Output&nbsp;</div>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/6.png" alt="" />&nbsp;Figure 2: Tool Output&nbsp;</div>
<p><br /><br /></p>
<ol start="3" type="1">
<li><strong>GO PubMed &ndash;</strong>It is a web server which allows users to explore PubMed search results with the Gene Ontology.</li>
</ol>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/7.png" alt="" /></div>
<p><br /><br /></p>
<div>The output page displays all the reference articles available for the protein.<br /><img src="http://pe-iitb.vlabs.ac.in/images/exp15/8.png" alt="" /></div>
<p><br /><br /></p>
<p>The most important columns in the profile.build() output are the II, XI and XII columns. The II column reports the code of the PDB sequence that was compared with the target sequence. The XI column reports the percentage sequence identities between betalactoglobulin and a PDB sequence normalized by the lengths of the alignment. In general, a sequence identity value above approximately 35% indicates a potential template. A better measure of the significance of the alignment is given in the XII column by the e-value of the alignment. To select the most appropriate template for our query sequence over the six similar structures, we will use the alignment.compare_structures() command to assess the structural and sequence similarity between the possible templates (file "betalactoglobulin_compare.py").</p>
<ol start="4" type="1">
<li><strong>GOanna</strong>GOanna is an annotation tool that allows users to find protein annotations, using a similarity search i.e. it conducts a BLAST search against a specific GO annotated database specified by the user and displays the BLAST results of the same.<br />Open the main page of the tool and adjust the following parameters are per the requirement. The input of the tool can be an accession number or the FASTA sequence. Results are sent by mail to the account provided at the time of submission.</li>
</ol>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/9.png" alt="" /></div>
<p><br /><br /></p>
<p>The output is mailed and usually in the excel format. The following is the output file which has columns displaying the best sequence homology entries with the query sequence.</p>
<p><br /><br /></p>
<div><img src="http://pe-iitb.vlabs.ac.in/images/exp15/10.png" alt="" /></div>
<p><br /><br /></p>
<p>Output Excel sheet</p>
<p></p>