<p>Bioinformatics is a combination of two major disciplines of science, the biological and computer sciences. The study carried over with respect to biology part is helped and represented in a better way with the help of computer tools together constitute bio-informatics. The need for bioinformatics arose with HumanGenome Project, which aimed to sequence the entire human genome. Today, bioinformatics has become indispensible to biological scientists and is involved in day to day research, since, it has grown to encompass proteomics, transcriptomics, molecular modeling and several other disciplines in the field of biological research. One such application by using bio-informatics tools has been explained in this module.&nbsp;<strong>Sequence Alignment</strong>&nbsp;is a process of aligning two sequences to achieve maximum levels of identity between them. The comparisons of the sequence are made with the query sequence to that of the database sequence. This help to determine the functional, structural and evolutionary relationships between them.</p>
<p>Information generated from sequence alignment can help in assigning functions to unknown protein, also determines the evolutionary relations of organisms and helps to predict the 3D structure of a protein.Homology is attributed to similarity due to a descent from a common ancestor, i.e. if two sequences are from different organisms are similar those are termed as Homologous.</p>
<p><br /><strong>Types Of Sequence Alignment&nbsp;</strong></p>
<p><strong>Based on sequence length :</strong>&nbsp;Depending on whether the sequences are compared in parts (similar sections) or as a whole, there are two methods:</p>
<p><strong>1. Global sequence alignment</strong>- In this alignment, the entire length of all the sequences are scanned to look for the similarities and gaps are inserted to fill unaligned spaces.1) Seq 1 -TAGC-GC-GT 2) Seq 2 - A-CA-CAGT</p>
<p><strong>2. Local sequence alignment&nbsp;</strong>&ndash; This alignment focuses on the local regions with high similarities and assigns gaps to portions that do not align.1) Seq 1 - CGATAACGTAT 2) Seq 2 - --ATAAAC---</p>
<p><strong>Based on number of sequences:</strong>According to number of sequence being compared there are two methods:</p>
<p><strong>1. Pairwise sequence alignment</strong>his involves aligning two sequences and to get the best region of similarity. 1) Seq 1 &ndash; 1 KTSSGNGAEDS 11 2) Seq 2 - 1 KTSSGNGAEDS 11</p>
<p><br /><br /></p>
<p><strong>Various methods used to perform pairwise alignment of nucleotide and protein sequences:</strong></p>
<p><strong>&Oslash; Dot Plot:</strong>Graphical method for two sequences to identify regions of similarity and dissimilarity represented with presence or absence of dots.</p>
<p><strong>&Oslash; Dynamic Programming:</strong>This method breaks a problem into small sub-problems and uses the solution of the sub-problems to compute the solution of the larger one. Some algorithms like Needleman-Wansch and Smith-Waterman are used here.</p>
<p><strong>&Oslash; Heuristic Method:</strong>When a single sequence is to be compared against the whole database, methods like BLAST and FASTA are used, which compare the query sequence one by one against all the sequences and then generates a result that shows the alignment of the query sequence with all matches in decreasing order of homology.</p>
<p><br /><strong>Some of the parameters used for producing optimum alignment are as follows-&nbsp;</strong></p>
<p><strong>&Oslash; Max target sequences :</strong>&nbsp;It displays the results with total number of aligned sequences on a page.</p>
<p><strong>&Oslash; Expected Threshold :</strong>&nbsp;It is a statistical indicator which calculates the probability that the resulting alignments are caused by random chance. The lower the E value, the more significant is the score. The default value is kept 10 as 10 matches are expected to be found random by chance (Stochastic model of Karlin &amp; Altschul, 1990).</p>
<p><strong>&Oslash; Query match :</strong>&ndash; It gives the maximum match in query range. This is useful for comparing many stronger matches of the query results from the weaker ones.</p>
<p><strong>&Oslash; Word size :</strong>&nbsp;This algorithm works by using word matches between the query and the database sequences. It searches for exact word matches, and then extends to the full length alignment. A word size of 3 is optimum for standard protein alignment and word size of 2 is required for short and nearly exact matches.</p>
<p><strong>Scoring schemes :</strong>&nbsp;Different scoring schemes algorithms are devised to obtain an optimum alignment. Use of any substitution matrix helps in aligning possible pairs of residues and also generates scores. To check the quality of pairwise sequence alignment, different PAM and BLOSUM matrices are used.</p>
<ul type="circle">
<li>PAM (Point Accepted Mutation) &ndash; This is developed by calculating the substitution of amino acid during evolution which are naturally accepted. PAM30 is used for sequence less than 35% in length whereas PAM70 is used for sequences ranging from 30% to 50%. (more details click here)</li>
<li>BLOSUM (Block amino acid substitution matrix) &ndash; This has been developed using conserved regions called BLOCKS, of distantly related protein sequences available within the block database. Out of all, BLOSUM 62 matrix is best used for detecting most protein similarities. BLOSUM 45 may be used for longer and weaker alignments. (more details click here)</li>
</ul>
<p><strong>&Oslash; Gap costs:</strong>A gap is a space which is introduced into an alignment to compensate for insertions and deletions in one sequence relative to another. Too many gaps should be avoided in the alignment and hence a gap penalty or gap score is assigned. Extension of the gap to encompass additional nucleotides or amino acid is also penalized in the scoring of an alignment. A penalty is subtracted from the alignment score when a gap penalty is introduced. Increase in gap costs parameter results in a lower number of gaps in the alignment. The penalty for the creation of a gap should be large enough so that gaps are introduced only where needed, and the penalty for extending a gap should take into account the likelihood that insertions and deletions occur over several residues at a time. Some commonly used values here are existence 10, 11 and extension 1.</p>
<center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/1.png" alt="" /></center>
<p><strong>2. Multiple sequence Alignment (MSA) -</strong>This involves the alignment of more than two (protein/ DNA) sequences and assesses the sequence conservation of proteins domains and structures. It is an extrapolation of the pairwise sequence alignment which reflects alignment of similar sequences and provides a better alignment score. Various analysis like Homology modeling for prediction of protein structure, Phylogenetic analysis, motif detection etc are based on the results of multiple sequence alignment. There are many softwares like Clustal, t-coffee, Phylip, MSA, MUSCLE used for obtaining multiple sequence alignment. Example &ndash;</p>
<ul type="circle">
<li>Seq 1 - PQGGGGWGQ</li>
<li>Seq 2 - PQGGGGWGQ</li>
<li>Seq 3 - PQGGGGWGQ</li>
<li>Seq 4 - PQGGGGWGQ</li>
<li>Seq 5 - PQGGGGWGQ</li>
</ul>
<p><br /><strong>Following parameters should be considered while aligning multiple sequences:</strong>&nbsp;</p>
<ul type="circle">
<li><strong>Protein weight matrix -</strong>The matrix used to generate the alignment score must be able to produce the highest score. Eg: PAM and BLOSUM.</li>
<li><strong>Gap open &ndash;</strong>The penalty to open a gap. The presence of a gap is frequently given more significance than the length of the gap. By default, the gap opening penalty is 10.</li>
<li><strong>Gap extension &ndash;</strong>&nbsp;The penalty to extend a gap. Extension of the gap also involves additional of amino acids which is penalized in the scoring of the alignment. By default, gap extension penalty is 0.20.</li>
</ul>
<p><br /><br /><strong>Application of MSA results:</strong></p>
<p><strong>Phylogenetic analysis &ndash;&nbsp;</strong>It is one of the major areas where multiple sequence analysis results are used to find the evolutionary relatedness between sequences. The results are displayed in form of phylogenetic tree which has set of nodes and branches to link the nodes.</p>
<p><br /><br /><strong>File format view:&nbsp;</strong></p>
<p><strong>PHYLIP -</strong>PHYLogeny Inference Package. It&rsquo;s a format for Joe Felsenstein&rsquo;s phylogenetic applications, having 8 letter maximum lengths for the sequence ID.</p>
<p><strong>Cladogram -&nbsp;</strong>In a cladogram, the external taxa line up neatly in a row. Their branch lengths are not proportional to the number of evolutionary changes and thus no evolutionary time analysis can be done. Only the relative ordering of the taxa can be analyzed.</p>
<center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/2.jpg" alt="" /></center>
<p><strong>Phylogram -</strong>In a phylogram, the branch lengths represent the amount of evolutionary divergence. Such trees are said to be scaled.</p>
<center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/3.jpg" alt="" /></center>
<p><strong>Pearson/FASTA -&nbsp;</strong>Text based format to represent amino acids in single letter code. It also has sequence names followed by comments.</p>
<center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/3.jpg" alt="" /></center><center><img src="http://pe-iitb.vlabs.ac.in/images/exp13/2.png" alt="" /></center>
<p>Consensus Symbol indication:</p>
<ol type="1">
<li>An * (asterisk) indicates positions which have a single, fully conserved residue.</li>
<li>A: (colon) indicates conservation between groups of strongly similar properties - scoring &gt; 0.5 in the PAM 250 matrix.</li>
<li>A. (Period) indicates conservation between groups of weakly similar properties - scoring less than = 0.5 in the PAM 250 matrix.</li>
</ol>
<p>The following is the color code of the amino acids according to the properties of the amino acids.</p>
<table border="">
<tbody>
<tr>
<td>Orange</td>
<td>G</td>
</tr>
<tr>
<td>Red</td>
<td>HKR</td>
</tr>
<tr>
<td>Blue</td>
<td>FWYM</td>
</tr>
<tr>
<td>GREEN</td>
<td>ILVTN</td>
</tr>
<tr>
<td>Yellow</td>
<td>P</td>
</tr>
</tbody>
</table>
<p><strong>Jalview -</strong>Java alignment editor. It is a visualization tool for alignment algorithms and other database search results.</p>
<p></p>